Lib/test/test_sys.py

```diff
@@ -1512,6 +1512,22 @@ def test_asyncgen_hooks(self):
         self.assertIsNone(cur.firstiter)
         self.assertIsNone(cur.finalizer)
 
+    def test_changing_sys_stderr_and_removing_reference(self):
+        # If the default displayhook doesn't take a strong reference
+        # to sys.stderr the following code can crash. See bpo-43660
+        # for more details.
+        code = """
```

Comment:
Could you please use textwrap.dedent() for that?

Reply:
Will do